packages/copyable/src/Copyable.tsx

```diff
@@ -160,23 +199,31 @@ interface CopyableProps {
 }
 
 export default function Copyable({
-  darkMode = false,
+  darkMode: darkModeProp,
   children,
   label,
   description,
   className,
   copyable = true,
-  size = Size.Default,
+  size,
   shouldTooltipUsePortal = true,
 }: CopyableProps) {
-  const mode = darkMode ? Mode.Dark : Mode.Light;
-
+  const { theme, darkMode } = useDarkMode(darkModeProp);
   const [copied, setCopied] = useState(false);
   const [showCopyButton, setShowCopyButton] = useState(false);
   const [buttonRef, setButtonRef] = useState<HTMLButtonElement>();
 
   const { portalContainer } = usePopoverPortalContainer();
 
+  // TODO: should API change to use number values instead of 'large and default' for component size?
+  // Normalize the size so that we can use the baseFontSize from the provider if the size is not set by the component
+  const normalizedSize = size
+    ? size === Size.Default
+      ? BaseFontSize.Body1
+      : BaseFontSize.Body2
+    : size;
```

Comment:
Looks like we're setting `normalizedSize = undefined` (or something else falsy) if `size` is falsy

Reply:
isn't that ok? override accepts: `BaseFontSize | 14 | undefined`. So if it's undefined then the baseFontSize will be returned?